app/controllers/publishers/vacancies_controller.rb

```diff
@@ -59,8 +61,25 @@ def summary
     @feedback_form = Publishers::JobListing::FeedbackForm.new
   end
 
+  def job_applications
+    @vacancy = VacancyPresenter.new(@vacancy)
+    @application_statuses = applications.group_by { |k| k[:status] }.transform_values(&:count)
```

Comment:
Can we live without this instance variable? In the template we might be able to do `job_applications.submitted.count` for example instead